src/Apps/Artwork/Components/OtherWorks/OtherWorksByArtist.tsx

```diff
@@ -0,0 +1,76 @@
+import { Flex } from "@artsy/palette"
+import { OtherWorksByArtist_artwork } from "__generated__/OtherWorksByArtist_artwork.graphql"
+import { OtherWorksByArtistQuery } from "__generated__/OtherWorksByArtistQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import ArtworkGrid from "Components/ArtworkGrid"
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { data as sd } from "sharify"
+import { Header } from "./Header"
+
+interface OtherWorksByArtistProps {
+  artwork: OtherWorksByArtist_artwork
+}
+
+export const OtherWorksByArtist: React.SFC<OtherWorksByArtistProps> = props => {
+  return (
+    <Flex flexDirection="column" alignItems="center">
+      <Header
+        title="Other works by Artist"
+        buttonHref={sd.APP_URL + props.artwork.artist.href}
+      />
+      <ArtworkGrid artworks={props.artwork.artist.artworks} />
+    </Flex>
+  )
+}
+
+export const OtherWorksByArtistFragmentContainer = createFragmentContainer(
+  OtherWorksByArtist,
+  graphql`
+    fragment OtherWorksByArtist_artwork on Artwork {
+      id
+      artist {
+        name
+        href
+        counts {
+          artworks(format: "0,0", label: "work")
+        }
+        # FIXME: add exclude: [$artistID]), but it throws relay compiler error
```

Comment:
When we were playing in GraphiQL, it looked like `exclude: $artistID` wound up working (ie- there was no validation error between single element vs an array), so maybe we can try that here?

Reply:
Ah this is copy paste from old PR -- no longer applies.